google-cloud-dlp/Rakefile

```diff
@@ -139,6 +139,24 @@ task :jsondoc => :yard do
             include: ["google/rpc"]
           }
         ]
+      },
+      {
+        type: "toc",
+        title: "Google::Privacy::Dlp::V2::DataTypes",
+        modules: [
+          {
+            title: "Google::Privacy::Dlp::V2",
+            include: ["google/cloud/dlp/v2"]
```

Comment:
The `include` pattern for the GAPIC code in "Data Types" needs to end in a slash `/`, or the output will include content from the `V2` module and end up broken with a table inside a table. The [Video Intelligence Rakefile](https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/master/google-cloud-video_intelligence/Rakefile#L132) does it correctly. Is this something that can be fixed in the generation? We just had this problem with BigQuery Data Transfer as well.

Line 131, although not part of this PR, needs the same fix, for `V2beta1`.

Reply:
Actually, @blowmage pointed out that other than Video Intelligence and BQ Data Transfer, most GAPIC packages do *not* have the trailing slash, and just show the module docs (the `V2` docs, in this case) at the Data Types page. Let's hold off on changing this for now and see what happens. If I get the chance, I'll try to run the docs locally to check them first, but it shouldn't hold up the PR.

Reply:
Whoops, just noticed your second comment. I'll revert it.

Reply:
Sorry about that!